.travis.yml

```diff
@@ -1,28 +1,26 @@
-sudo: false
 language: ruby
 cache: bundler
 rvm:
   - 2.6.3
 
-before_install:
-  - sudo apt-get update
-  - sudo apt-get --yes remove postgresql\*
-  - sudo apt-get install -y postgresql-12 postgresql-client-12
-  - sudo cp /etc/postgresql/{9.6,12}/main/pg_hba.conf
-  - sudo service postgresql restart 12
-  - gem install bundler
+services:
```

Comment:
are all these instructions needed? Here for example we didn't need to add any of that https://github.com/rootstrap/active-storage-base64/blob/master/.travis.yml

Reply:
Is needed for the line 21. The `structure.sql` is for postgres 12, and travis default postgres version is 9, so these lines are to set to 12